client/wfnews-war/src/main/angular/src/app/components/full-details/area-restrictions-full-details/area-restrictions-full-details.component.ts

```diff
@@ -1,5 +1,5 @@
 import { ChangeDetectorRef, Component, Input, OnInit } from '@angular/core';
-import { Router as Route } from '@angular/router';
+import { Router as Route, Router } from '@angular/router';
```

Comment:
This looks like a double import to me? Router is already imported

Reply:
good catch